### ISSUE TITLE
Make theme for black-and-white plots

### ISSUE THREAD

**Issue by stefan-mueller** [fsolt/dotwhisker#71]
I was thinking whether we might add an option to create a black-and-white plots. I know that it's possible to adjust it with `theme_`, but this can be a bit tedious. A simple example:

```r
library(dotwhisker)

m1 <- lm(mpg ~ wt + cyl + disp + gear, data = mtcars)

# draw a dot-and-whisker plot
dwplot(m1)

# apply theme_bw() from ggplot2
dwplot(m1) + 
  theme_bw()
```

For the second plot, the bar colours are in still red, and the legend "model" is added although we have only one model.

I was thinking about adding an option called, for example, `theme = "bw"`, which takes care of this. By default in this theme, we could 
- use **ggplot2**'s `theme_bw()`,
- change the axis labels from grey to black,
- disable the legend when we plot only one model,
- use `scale_colour_grey()` by default, and/or use different shapes for the point estimates to distinguish between models when we have a list of models to be plotted.

Adjustments could still be made by adding/overwriting the default of `theme = "bw"`by using **ggplot2** options.

Basically, these plots would look similar to the black and white ones from the [vignette](https://cran.r-project.org/web/packages/dotwhisker/vignettes/dotwhisker-vignette.html).

What are your thoughts on this? Feedback welcome. I'm happy to work on this.

**Comment by stefan-mueller** [comment]
#78 pretty much solves this feature request, so I close this issue. Thanks for the recent improvements by the way, @fsolt!

**Comment by fsolt** [comment]
Getting rid of cowplot (which I did to close #78) means the default dotwhisker plot isn't black-and-white anymore, so I guess this one should be reopened.  The "disable the legend when we plot only one model" bullet was included at some point, but the rest remains.

My feeling about this, tbh, is that theme formatting such as this should probably remain out of the package if it can be accomplished via ggplot2 calls after the dotwhisker plot is made (that's true here, right?), but I'm open to arguments to the contrary . . .

**Comment by stefan-mueller** [comment]
I agree, we can close this issue. Users can simply use **ggplot2** options to customise the theme (as we show in the vignette).
